tests/test_backends.py

```diff
@@ -283,3 +287,38 @@ def test_build_vllm_cmd_with_args_provided(tmp_path: pathlib.Path):
         host, port, model_family, model_path, str(chat_template), vllm_args
     )
     assert cmd == expected_cmd
+
+
+def test_build_vllm_cmd_with_bnb_quant(tmp_path: pathlib.Path):
+    host = "localhost"
+    port = 8080
+    model_path = tmp_path / "model"
+    model_family = ""
+    chat_template = tmp_path / "chat_template.jinja2"
+    chat_template.touch()
+    vllm_args: list[str] = []
+    expected_cmd = [
+        sys.executable,
+        "-m",
+        "vllm.entrypoints.openai.api_server",
+        "--host",
+        host,
+        "--port",
+        str(port),
+        "--model",
+        str(model_path),
+        "--chat-template",
+        str(chat_template),
+        "--quantization",
+        "bitsandbytes",
+        "--load-format",
+        "bitsandbytes",
+        "--enforce-eager",
+        "--distributed-executor-backend",
+        "mp",
+    ]
+    create_safetensors_or_bin_model_files(model_path, "safetensors", True)
+    cmd, _ = build_vllm_cmd(
+        host, port, model_family, model_path, str(chat_template), vllm_args
+    )
+    assert cmd == expected_cmd
```

Comment:
This is a bit fragile: every time some other argument will be added to the default command line, this test will have to be updated. Consider instead confirming that the exact three options you test here are present (and not that the whole command line matches.)